util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+
```

Comment:
Also, this script doesn't belong in the `util/` directory.  There isn't a perfect place for it, but `cmd/fscrypt/` would be best I think.

Reply:
Moved.

Should that script be installed as `$PREFIX/share/bash-completion/completions/fscrypt` while running `make install`?

Reply:
Yes, I think `make install` should install this.

Reply:
Can you make `make uninstall` delete it, as well?

Reply:
Done.